lib/initInstance.js

```diff
@@ -132,16 +132,24 @@ function generateComponentJson() {
 }
 
 
-let instanceItems = [];
-try {
-  instanceItems = fs.readdirSync(config.pluginsDir);
-} catch (e) {
-  console.warn("ZWED5003W - Warning: couldn't read plugin directory",e);
-  //Couldnt read, will copy defaults
+function getPluginJsonNames() {
+  try {
+    return fs.readdirSync(config.pluginsDir);
+  } catch (e) {
+    console.warn("ZWED5003W - Warning: couldn't read plugin directory",e);
+  }
+  return [];
 }
+
+let instanceItems = getPluginJsonNames();
 //Copy default plugins if could not find configjs - implies something wrong with environment.
 if (instanceItems.indexOf('org.zowe.configjs.json') == -1) {
   initUtils.registerBundledPlugins(config.pluginsDir, instancePluginStorage, instanceItems, FILE_MODE);
+  instanceItems = getPluginJsonNames();
+  if (instanceItems.indexOf('org.zowe.configjs.json') == -1) {
+    console.warn('ZWED0156E - Could not register default plugins into app-server');
```

Comment:
Don't you need to add log message to 
`zlux-app-manager\bootstrap\src\assets\i18n\log\messages_en.json` ?

Reply:
No nvm: https://github.com/zowe/zlux-server-framework/pull/252/files